ci/do_ci.sh

```diff
@@ -59,7 +59,11 @@ mkdir -p "${BUILD_DIR}"
 [ -z "${PLUGIN_DIR}" ] && export PLUGIN_DIR=$HOME/plugin
 mkdir -p "${PLUGIN_DIR}"
 
-BAZEL_OPTIONS="--copt=-DENABLE_METRICS_PREVIEW --copt=-DENABLE_LOGS_PREVIEW --copt=-DENABLE_TEST"
+BAZEL_OPTIONS="--copt=-DENABLE_LOGS_PREVIEW --copt=-DENABLE_TEST"
+# Previous legacy metrics use virtual drive, which can not be used without RTTI
+if [[ "$1" != "bazel.nortti" ]]; then
+  BAZEL_OPTIONS="$BAZEL_OPTIONS --copt=-DENABLE_METRICS_PREVIEW"
```

Comment:
Is there code in _metrics api/sdk which can't be used with the static cast?

Reply:
> Is there code in _metrics api/sdk which can't be used with the static cast?

Yes, it use virtual inheritance.That's why I create #1310 before.